accept.go

```diff
@@ -150,12 +152,34 @@ func accept(w http.ResponseWriter, r *http.Request, opts *AcceptOptions) (_ *Con
 	b, _ := brw.Reader.Peek(brw.Reader.Buffered())
 	brw.Reader.Reset(io.MultiReader(bytes.NewReader(b), netConn))
 
+	ipStr, portStr, err := net.SplitHostPort(r.RemoteAddr)
```

Comment:
A couple of notes:

- Parsing, and thus erroring here could potentially cause issues for current users (new behavior)
- We could call `netConn.RemoteAddr()` instead in our `RemoteAddr` func